src/inmanta/app.py

```diff
@@ -83,6 +83,7 @@
 
 @command("server", help_msg="Start the inmanta server")
 def start_server(options: argparse.Namespace) -> None:
+    LOGGER.info(f"Starting inmanta-server version {__version__}")
```

Comment:
This will log the version of the inmanta-core component. The idea was to log the version of the product.